packages/web/src/components/collection/desktop/ViewerActionButtons.tsx

```diff
@@ -18,21 +18,19 @@ export const ViewerActionButtons = (props: ViewerActionButtonsProps) => {
     getCollection(state, { id: collectionId })
   ) as Collection | null
 
-  const { track_count, is_private, access } = collection ?? {}
-  const isDisabled = !collection || track_count === 0 || is_private
+  const { track_count, is_private: isPrivate, access } = collection ?? {}
+  const isDisabled = !collection || track_count === 0 || isPrivate
   const hasStreamAccess = access?.stream
 
-  return (
+  return isPrivate ? null : (
```

Comment:
nit: I feel like I usually see `check ? stuff : null` in our code base, but either is fine

idk which I personally like better, but I see that more often